### ISSUE TITLE
move mouse

### ISSUE THREAD

**Issue by Oxenso** [kenn-notation/Valorant-External#14]
How do I move the mouse, as I see the code in the driver is already there, but it apparently does not work, can you help me?
example:
`driver.move_mouse(Width / 2, Height / 2);`

(if anything the ESP itself works fine!)

**Comment by AvileForsure** [comment]
it does apparently work but not in valorant. They possibly block it.

**Comment by quantumwolf-git** [comment]
Yeah I just went ahead and bought one. Been working like a charm since
